# Hekateros hekateros_control/IsDescLoaded service

---
bool is_desc_loaded   # specific hekateros model is [not] loaded and parsed
